Tests/SecureXPCTests/XPCEncoder/Scalar encoding tests.swift

```diff
@@ -0,0 +1,240 @@
+		try assert(-123 as Int32, encodesEqualTo: xpc_int64_create(            -123))
+		try assert(   0 as Int32, encodesEqualTo: xpc_int64_create(               0))
+		try assert( 123 as Int32, encodesEqualTo: xpc_int64_create(             123))
+		try assert(.max as Int32, encodesEqualTo: xpc_int64_create(Int64(Int32.max)))
+	}
+	
+	func testEncodes_Int64_as_XPCInt64() throws {
+		try assert(.min as Int64, encodesEqualTo: xpc_int64_create(Int64(Int64.min)))
+		try assert(-123 as Int64, encodesEqualTo: xpc_int64_create(            -123))
+		try assert(   0 as Int64, encodesEqualTo: xpc_int64_create(               0))
+		try assert( 123 as Int64, encodesEqualTo: xpc_int64_create(             123))
+		try assert(.max as Int64, encodesEqualTo: xpc_int64_create(Int64(Int64.max)))
+	}
+	
+	// MARK: Unsigned Integers
+	
+	func testEncodes_UInt_as_XPCUInt64() throws {
+		try assert(.min as UInt, encodesEqualTo: xpc_uint64_create(UInt64(UInt.min)))
+		try assert(   0 as UInt, encodesEqualTo: xpc_uint64_create(               0))
+		try assert( 123 as UInt, encodesEqualTo: xpc_uint64_create(             123))
+		try assert(.max as UInt, encodesEqualTo: xpc_uint64_create(UInt64(UInt.max)))
+	}
+	
+	func testEncodes_UInt8_as_XPCUInt64() throws {
+		try assert(.min as UInt8, encodesEqualTo: xpc_uint64_create(UInt64(UInt8.min)))
+		try assert(   0 as UInt8, encodesEqualTo: xpc_uint64_create(                0))
+		try assert( 123 as UInt8, encodesEqualTo: xpc_uint64_create(              123))
+		try assert(.max as UInt8, encodesEqualTo: xpc_uint64_create(UInt64(UInt8.max)))
+	}
+	
+	func testEncodes_UInt16_as_XPCUInt64() throws {
+		try assert(.min as UInt16, encodesEqualTo: xpc_uint64_create(UInt64(UInt16.min)))
+		try assert(   0 as UInt16, encodesEqualTo: xpc_uint64_create(                 0))
+		try assert( 123 as UInt16, encodesEqualTo: xpc_uint64_create(               123))
+		try assert(.max as UInt16, encodesEqualTo: xpc_uint64_create(UInt64(UInt16.max)))
+	}
+	
+	func testEncodes_UInt32_as_XPCUInt64() throws {
+		try assert(.min as UInt32, encodesEqualTo: xpc_uint64_create(UInt64(UInt32.min)))
+		try assert(   0 as UInt32, encodesEqualTo: xpc_uint64_create(                 0))
+		try assert( 123 as UInt32, encodesEqualTo: xpc_uint64_create(               123))
+		try assert(.max as UInt32, encodesEqualTo: xpc_uint64_create(UInt64(UInt32.max)))
+	}
+	
+	func testEncodes_UInt64_as_XPCUInt64() throws {
+		try assert(.min as UInt64, encodesEqualTo: xpc_uint64_create(UInt64(UInt64.min)))
+		try assert(   0 as UInt64, encodesEqualTo: xpc_uint64_create(                 0))
+		try assert( 123 as UInt64, encodesEqualTo: xpc_uint64_create(               123))
+		try assert(.max as UInt64, encodesEqualTo: xpc_uint64_create(UInt64(UInt64.max)))
+	}
+	
+	// MARK: Floating point numbers
+	
+	func testEncodes_Float_as_XPCDouble() throws {
+		// "Normal" values, from lowest to highest
+		try assert(-Float.infinity,
```

Comment:
Minor, but the indention used for all of the second parameters here strikes me as rather odd as doesn't appear to be aligned with anything in particular. I'm used to aligning all of the parameters to have the same indentation. Is there a specific formatting convention you follow/are used to?

Reply:
I think it started out with Xcode aligning it like 

```
assert(foo
       bar: ...)
```

And then I messed it up from there. I'll lint it all before merging.